client/apps/admin/components/application_instances/index.jsx

```diff
@@ -45,6 +45,8 @@ export class Index extends React.Component {
     this.state = {
       modalOpen: false,
       currentPage: null,
+      column: 'created_at',
```

Comment:
use sortColumn instead of column to make it clear what the value is for.
```suggestion
      sortColumn: 'created_at',
```